front/src/hooks/useNavigation.ts

```diff
@@ -20,13 +20,15 @@ export const useNavigation = () => {
         for (const [key, value] of Object.entries(FilterBy)) {
             params.set(key, value);
         }
-        if (safeCastToTimeOption(params.get("dateTimeFilter"))) {
+
+        const isCustomInterval = FilterBy.endDateTime && FilterBy.startDateTime;
+        if (isCustomInterval)
+            params.delete("dateTimeFilter");
```

Comment:
(refactor) Is "dateTimeFilter" used elsewhere? could we have it in a file somewhere as a const if so?